### PR TITLE
Feat: tf2.4 bazel build file

### LINKED ISSUE #245: TensorFlow 2.4 package
As of today, (2021-02-02), there is no build / package for TensorFlow 2.4. The most recent minor version of TensorFlow that is supported / built is [2.3](https://github.com/waymo-research/waymo-open-dataset/blob/e307878903620af93ad083e47be8412d505d46a6/tf/workspace_tf2_3_0.bzl). Do you have any plans of building a version for 2.4?
How hard would it to contribute that, if you don't have time?
Final question :laughing:, why do you have to maintain separate packages? I couldn't find anything in the readme.

### PR DESCRIPTION
**Pull request by patzm**
Closes #245
Yo, the build worked locally. I haven't tested it beyond that though. Do you have automated tests for that? Or is a succeeding build sufficient?
Either way, you would still need to push / publish the new package.

### CODE REVIEW

**Comment by patzm** [comment]
@googlebot I fixed it.

File: tf/workspace_tf2_4_0.bzl [hunk 1]
```diff
@@ -0,0 +1,51 @@
+"""Install dependencies."""
```
Comment (peisun1115) [inline tf/workspace_tf2_4_0.bzl:1]: can you do git cp so that we can see diffs between different version?

**Review by peisun1115 — COMMENTED** [review]
Thanks for contributing

File: tf/workspace_tf2_4_0.bzl (hunk 1 above)
Comment (patzm) [reply tf/workspace_tf2_4_0.bzl:1]: Sorry, I actually never used that and can't find anything apart from this [SO post](https://stackoverflow.com/questions/47401843/git-copy-file-as-opposed-to-git-mv). But I just ran this locally:
```bash
diff workspace_tf2_3_0.bzl workspace_tf2_4_0.bzl
```
with an empty output, so the files are indeed identical.

**Comment by patzm** [comment]
Merge conflicts are resolved in 81f57904ebeb920dc183642217548ae126cef405.

**Review by peisun1115 — APPROVED** [review]

**Comment by patzm** [comment]
@peisun1115, do you know about the PyPI package? I just checked and it hasn't been released yet for 2.4. Who usually does that?

**Comment by peisun1115** [comment]
I usually do it but i haven't got time to do so.  i assume you can simply upload one on your own or build one locally if you need?  probably use a different name if you want to upload on your own to avoid name collisions with our updates

**Comment by patzm** [comment]
Yes I could do that, but I think this will just confuse people. For now, I build locally and wait for you to push it. Thx for the quick reply. Pls if possible, ping me quickly here once you have pushed the package :pray :wink:

**Pull request merged**